### PR TITLE
Move Voting Support under How To Claim Your Vote #88

### PR DESCRIPTION
**Pull request by VenomIreland**
Re-ordered div so that "Voting Support" shows up under "How To Claim Your Vote". Added a blank div to force "Can I Help Some Other Way?" to the left side of the page, however, this blank div sometimes shows up underneath "Can I Help Some Other Way?" and I am unsure how to remedy that. Without the blank div, the countdown clock shows up on the left side, leaving a lot of empty space in the right column.

### CODE REVIEW

**Comment by VenomIreland** [comment]
If line 524 is changed from 
`<div class="bannerbox" id="blank">`
to
`<div>`
the minor issue I mentioned above no longer appears. Apologies, this is my first Pull Request so I am unsure how to add a new commit to this.

**Comment by jkbits1** [comment]
thanks for your PR :+1: someone will take a look today or tomorrow

if you add a new commit to your `reorder-driv` branch, that change should appear in this PR :smile:

File: index.html [hunk 1]
```diff
@@ -522,6 +509,20 @@ <h3 class="bannerbox__title">Can I help some other way?</h3>
...
+                        <p>The <a href="http://www.866ourvote.org/">Election Protection Coalition</a> works around the clock to advance and defend your right to vote.</p>
+                        <p>Report voting problems, and get voting relating info for your state.</p>
+                        <ul>
+                            <li>Visit <a href="http://www.866ourvote.org/">866ourvote.org</a></li>
+                            <li>Call <a href="tel:8666878683">866-OUR-VOTE</a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div>                   
+                </div>
```
Comment (richardwestenra) [inline index.html]: That's a clever solution, I wouldn't have thought of it. It's not ideal in terms of adding unnecessary markup, but if gets the job done 👍

File: index.html [hunk 2]
```diff
@@ -254,7 +254,7 @@ <h3 class="bannerbox__title">I need a ride</h3>
                                 <label for="RiderWillBeSafe">
                                     <input type="checkbox" id="RiderWillBeSafe" name="RiderWillBeSafe" required /> I understand that Carpool Vote provides introductions between riders and volunteer drivers who have signed up on the platform. I understand that anybody can sign up to drive and Carpool Vote is unable to perform any background checks on people who use the platform. As with any other environment where I meet new people, I will take steps to keep myself and my possessions safe and accept that Carpool Vote cannot be responsible if anything goes wrong.
                                 </label>
-								<div class="help-block with-errors"></div>
+                                <div class="help-block with-errors"></div>
```
Comment (richardwestenra) [inline index.html]: Good catch, thanks 👍

**Pull request merged**